### ISSUE TITLE
ARKs are incorrect for YMDI material

### ISSUE THREAD

**Issue by joycebcat** [pulibrary/figgy#1741]
I have had a message from a scholar working with the YMDI (Yemeni manuscripts).  She has been accessing them through our catalog.  The ARKs seem to have gotten messed up with the migration to DPUL so what is showing in the BL catalog isn't always the right set of images.

Figgy has the correct images for the metadata:
figgy: https://figgy.princeton.edu/catalog/28cfa04a-b699-427c-84e6-998b74f9669e
source id: 7849027 
local identifier: pkw52kb38t 
identifier: ark:/88435/cj82k733w  -- this ARK doesn't lead to this object, see below

The source bib record has different images:
https://catalog.princeton.edu/catalog/7849027#view
http://arks.princeton.edu/ark:/88435/qv33rx40r 

The ARK in the bib and from figgy no longer match.  When I try to search for the figgy ARK (as below), I get to a different bib record in BL with the same images--but this time the images and the MD match in BL, but don't match the figgy data.  The researcher has found 3 different BL records which lead to these same images.
http://arks.princeton.edu/ark:/88435/cj82k733w

It seems like the ARK assignment has several ARKs pointing to the same digital images.  Figgy has a correct MD and image connection, but the ARKs showing up in the MD aren't all pointing to the right place.

**Comment by tpendragon** [comment]
@joycebcat Yup, it looks like every Yemeni object has the same ARK in Figgy. Probably importing the wrong one? Very strange.

**Comment by tpendragon** [comment]
Okay. I got this fixed. For the record:

1. Fixed marc_liberation so it would prioritize an ark with a known Figgy identifier for future record imports. This made it choose the right ID for refreshing metadata. (https://github.com/pulibrary/marc_liberation/pull/452)
2. For every Yemeni item, delete the `identifier` property off the ScannedResource and refresh the remote identifier so it'll pick up the one from the MARC record and update that ARK to point to the correct catalog page.
3. Reindex every Yemeni item in Orangelight
